test/catch2_test_block_shuffle.cu

```diff
@@ -120,11 +122,19 @@ struct rotate_op_t
 template <class T>
 struct up_with_suffix_op_t 
 {
-  int m_target_thread_id;
```

Comment:
Let's keep a single variable here and compute it's value using `cub::RowMajorTid`.